src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20240716123651_EES5170_AddBoundaryDataTablePermission.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
+{
+    /// <inheritdoc />
+    public partial class EES5170_AddBoundaryDataTablePermission : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("" +
+                "IF EXISTS (SELECT name " +
```

Comment:
Happy with the IF EXISTS, but I don't think you actually need it.  We always have it locally and in the Azure environment.

Reply:
I'll leave it for now, no harm in an extra `SqlExcepion` failsafe!